python/ray/train/batch_predictor.py

```diff
@@ -136,6 +141,16 @@ def predict(
             Dataset containing scoring results.
 
         """
+        if num_gpus_per_worker is None:
+            num_gpus_per_worker = 0
+        if num_cpus_per_worker is None:
+            if num_gpus_per_worker > 0:
```

Comment:
should we be doing this across all of our library code 😅

Reply:
Yeah, it's probably a good idea.